fiftyone/server/query.py

```diff
@@ -135,7 +135,7 @@ def from_doc(cls, doc: SavedViewDocument):
         stage_dicts = [json_util.loads(x) for x in doc.view_stages]
         saved_view = from_dict(
             data_class=cls,
-            data=doc.serialize(),
+            data=doc.to_mongo(),
```

Comment:
Use `to_dict()` here. The goal of the `fiftyone.core.odm.XXXDocument` classes is to abstract the "mongo" part away from the rest of the code. Ideally in the future we could swap out mongo for something else and the code would continue to work. In order to achieve that, we should only be calling methods that are explicitly defined and documented on the `fiftyone.core.odm.XXXDocument` classes